R/translate-sql-window.r

```diff
@@ -194,6 +197,14 @@ win_current_frame <- function() sql_context$frame
 # Not exported, because you shouldn't need it
 sql_current_con <- function() sql_context$con
 
+# Functions to manage information for special cases
+set_current_context <- function(context) {
+  old <- sql_context$context
+  sql_context$context <- context
+  invisible(old)
+}
+
+sql_current_context <- function() sql_context$context
```

Comment:
It should be documented in the same file as all the other `sql_current_` functions